karma.conf.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+
+var path = require('path');
+
+
+var webpackConfig = {
+  devtool: 'eval',
+  resolve: {
+    extensions: ['', '.js'],
+  },
+  module: {
+    loaders: process.env.COVERAGE ?
+      [
+        {test: /\.js$/, loader: 'babel', include: [path.resolve('./test')]},
+        {test: /\.js$/, loader: 'isparta', include: [path.resolve('./src')]},
```

Comment:
This is necessary to get proper ES6 code coverage
<img width="1183" alt="20150706-180137" src="https://cloud.githubusercontent.com/assets/175264/8517961/18805332-2409-11e5-8858-e316d8587b6b.png">